app/src/main/java/ciserver/ContinuousIntegrationServer.java

```diff
@@ -47,6 +52,9 @@ private void handlePushEvent(PushEvent event) {
         System.out.println("Commit status update sent successfully.");
     }
 
+    /**
+     * {@inheritDoc}
```

Comment:
This might not be sufficient, perhaps we should write our own (at leat partially) custom description instead of just inheriting from AbstractHandler.handle()

Reply:
For `@{inheritDoc}` to be able to find what to copy we would need the [source available at compile time](https://docs.oracle.com/javase/1.5.0/docs/tooldocs/windows/javadoc.html#inheritingcomments)

Might be better to use `@see` and link [to this url](https://www.eclipse.org/jetty/javadoc/jetty-9/org/eclipse/jetty/server/handler/AbstractHandler.html#handle(java.lang.String,org.eclipse.jetty.server.Request,javax.servlet.http.HttpServletRequest,javax.servlet.http.HttpServletResponse))